pkg/provider/azure_loadbalancer_backendpool.go

```diff
@@ -185,8 +185,13 @@ func (bc *backendPoolTypeNodeIPConfig) ReconcileBackendPools(clusterName string,
 				for _, ipConf := range *bp.BackendIPConfigurations {
 					ipConfID := to.String(ipConf.ID)
 					nodeName, _, err := bc.VMSet.GetNodeNameByIPConfigurationID(ipConfID)
-					if err != nil && !errors.Is(err, cloudprovider.InstanceNotFound) {
-						return false, false, err
+					if err != nil {
```

Comment:
Should such change be applied to L90 as well?
https://github.com/kubernetes-sigs/cloud-provider-azure/blob/f782e89a13969aa970838719ac8a5f6c93dc7403/pkg/provider/azure_loadbalancer_backendpool.go#L90

Reply:
It doesn't have to. That func is for multi-slb and no matter single or multi-slb is used, ReconcileBackendPool will be invoked.